src/app/public/modules/text-editor/menubar/text-editor-menubar.component.scss

```diff
@@ -0,0 +1,30 @@
+.sky-text-editor-dropdown-item-divider {
+  border: 0;
+  padding: 0;
+  height: 1px;
+  margin: 9px 1px;
+  overflow: hidden;
+  background: transparent;
+  border-bottom: 1px solid rgba(0,0,0,0.1);
+  cursor: default;
+  filter: none;
+}
+
+.sky-font-deemphasized,
```

Comment:
Should we try to future-proof this a little bit? I know that this works today - but are we always guaranteed that all deemphasized things here will have this? Should we add some outer classes and lock this down to the two places it applies today?

Reply:
Especially with the fact that view encapsulation is turned off in this component. We risk affecting other non-text editor deemphasized text with it off and these not being wrapped in any way.

Reply:
Totally agree! Updated.